R/raster-methods.R

```diff
@@ -12,14 +12,15 @@ setOldClass("sf")
 #' @aliases raster
 #' @importFrom sp CRS
 #' @importFrom raster origin extend crop extent
+#' @importFrom sf st_crs
```

Comment:
The `sf` package is listed under Suggests in the DESCRIPTION, so essential functions shouldn't rely on it. See [9e8118](https://github.com/FlukeAndFeather/fasterize/commit/9e8118a55023aa0e044c34487725fd2326733c9a) for a different solution.